python-avd/pyavd/_eos_designs/structured_config/base/__init__.py

```diff
@@ -234,20 +234,16 @@ def hardware(self) -> dict | None:
     @cached_property
     def daemon_terminattr(self) -> dict | None:
         """
-        daemon_terminattr set based on cvp_instance_ip and cvp_instance_ips variables.
+        daemon_terminattr set based on cvp_instance_ips.
 
         Updating cvaddrs and cvauth considering conditions for cvaas and cvp_on_prem IPs
 
-            if 'arista.io' in cvp_instance_ip:
+            if 'arista.io' in cvp_instance_ips:
```

Comment:
i dont think here cvp_instance_ip is deprecated one, its the element of cvp_instance_ip_list so no changes require for this line

Reply:
For pseudocode I think the change is more useful that using the variable name internally to this function